prerequisites help.md

```diff
@@ -0,0 +1,55 @@
+# 🤖 **Help with the prerequisites** 🤖
+
+Naughtify requires some prerequisites that not everyone is familiar with. Here are some approaches that may help you.
+
+1. VPS - Virtual Private Server
+2. Domain - Your Link to the Server 
+3. LNbits - The Wallet invoice key and lighting address
+4. Telegram Bot - Botfather and bot token
+5. UserInfoBot - Your User / Chat ID
+
+### 1. VPS - Virtual Private Server
+
+If you don't have your own LNbits server that is publicly accessible, then you can rent a small VPS. The smallest version with 1 CPU and 512 MB RAM should be enough. You can find out how to set up a VPS here, for example, under [7.1 - General - Setting up a Virtual Private Server (VPS)](https://ereignishorizont.xyz/lnbits-server/en/#71_-_General_-_Setting_up_a_Virtual_Private_Server_VPS). 
+
+### 2. Domain - Your Link to the Server 
+
+If you host your own LNbits server publicly, you also have your own domain. You can set up additional subdomains with domain hosting providers. If you don't have your own domain and use someone else's LNbits server, you can use [duckdns.org](https://www.duckdns.org/). There you can get a free subdomain in the format `mysubdomain.duckdns.org`. All you have to do is point the domain to the IP address of the VPS on which Naughtify is installed and set up Caddy accordingly. That's it. 
+
+### 3. LNbits - The Wallet invoice key and lighting address
+
+LNbits is a wallet and account system. Each account can have multiple wallets and add extensions for different functions. Below is a brief explanation of how to create a wallet, receive the wallet invoice key and generate a static LNURLpay link with a Lightning⚡ address.
+
+3.1 Create an account with an LNbits server of your choice
+
+E.g. at the official demo server [demo.lnbits.com](https://demo.lnbits.com) or at LNbits.de [LNbits.de](https://LNbits.de). 
```

Comment:
Please change to this:

E.g. at the official demo server [demo.lnbits.com](https://demo.lnbits.com) or at community driven TimeCatcher Server [timecatcher.lnbits.de](https://timectcher.lnbits.de).

Reply:
done